### ISSUE TITLE
video and screen share does not work on 1 system

### ISSUE THREAD

**Issue by gsudhanshu** [aws/amazon-chime-sdk-js#1486]
### What happened and what did you expect to happen?

we are using aws chime sdk js for our meeting and conference application. On one particular system the video and screen share would not work.

System - win10, chrome browser

logs attached:
main.3d338280260dc6198c78.js:1 2021-08-03T10:52:34.033Z [ERROR] MyLogger - failed to connect

### Have you reviewed our existing documentation?

- [X] GitHub [issues](https://github.com/aws/amazon-chime-sdk-js/issues)
- [X] FAQs [page](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html)
- [X] README [page](https://github.com/aws/amazon-chime-sdk-js#amazon-chime-sdk-for-javascript)
- [X] Feature-specific [guides](https://github.com/aws/amazon-chime-sdk-js#guides)

### Reproduction steps

join meeting from 2 system

video does not work on 1 system

### Amazon Chime SDK for JavaScript version

2.4.1

### What browsers are you seeing the problem on?

chrome

### Browser version

67

### Browser console logs

[logs](https://drive.google.com/file/d/1jNe5VV5go0xKiEf45RG9hglyCkoirjXd/view?usp=sharing)

**Comment by devalevenkatesh** [comment]
Hello,

Thanks for reporting the issue. We see that you are using a very old version of Amazon Chime SDK for JavaScript as well as the Chrome browser.

You are running into Websocket connection failures on that specific device.
```
main.3d338280260dc6198c78.js:1 WebSocket connection to 'wss://signal.m2.as1.app.chime.aws/control/9c7472ac-34cc-40cb-8c57-f33ac1410706?X-Chime-Control-Protocol-Version=3&X-Amzn-Chime-Send-Close-On-Error=1&X-Amzn-Version=2.4.1&X-Amzn-User-Agent=chrome-92' failed: 
.
.
.
main.3d338280260dc6198c78.js:1 2021-08-03T10:52:34.033Z [INFO] MyLogger - notifying event: WebSocketFailed
main.3d338280260dc6198c78.js:1 2021-08-03T10:52:34.033Z [WARN] MyLogger - stopped pinging (WebSocketFailed)
```
If this is a wider impact, the issue should be appearing on multiple devices as well, please let us know if that is the case. Further, check if you are using any middleware in you application which adds some content security while making network requests or webcocket connection open requests as was one of the problem reported in #483 [here](https://github.com/aws/amazon-chime-sdk-js/issues/483#issuecomment-731268633).

We have an FAQ on the Networking setup for Chime request you to go through it once:
https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#networking

We only officially support Chrome browser 78+. Request you to check the browser support at below link and request you to update the JS SDK to a more recent version. There would be no breaking changes from 2.4.1 to latest (2.14.0) as the major version is still the same.
https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers

Please let us know if you are still running into the issue after upgrading to newer versions as suggested. Closing the issue for now as we do not support Chrome 67, but feel free to re-open or create a new one once you upgrade and run into the same problem.

Few other Websocket connection issues received in the past, going through them might help as well:
#492 , #483

**Comment by gsudhanshu** [comment]
@devalevenkatesh 

the browser used was : Google Chrome Version 92.0.4515.131 (Official Build) (64-bit)

sorry for missinformation earlier

**Comment by gsudhanshu** [comment]
how to reopen this issue?

**Comment by devalevenkatesh** [comment]
@gsudhanshu thanks for correcting it, checking if you went through other issues and FAQ I linked?

**Comment by gsudhanshu** [comment]
Yes have gone through issues pointed out.. but none solves my problem

**Comment by devalevenkatesh** [comment]
Could you please check if you have added and blocked "app.chime.aws" if you have set the Content Security Policy in the HTML?

Are there any additional network settings on this system vs the rest that are working? Generally the WebSocket connection issues are due to firewall or network settings, and weirdly if other systems are working and only this system is having problem then there must be some configuration on that system which is not able to connect.

**Comment by gsudhanshu** [comment]
Can you kindly check logs and try to see where the problem is?

**Comment by devalevenkatesh** [comment]
I have actually went through the logs and as commented in the first comment we see WebSocket failure only. 
Comment link: https://github.com/aws/amazon-chime-sdk-js/issues/1486#issuecomment-893004654

Could you please provide more logs if earlier attached logs have changed?
